contrib/node/examples/src/node/yarn-workspaces/BUILD

```diff
@@ -0,0 +1,23 @@
+node_module(
+  sources=rglobs('package.json', 'yarn.lock', 'adder/*', 'add-two/*'),
```

Comment:
This being recursive is going to result in this target owning multiple `package.json` files. If you only wanted to be recursive with respect to `adder` and `add-two`, then you'd want to use `adder/**`, etc (we use git syntax, so a double glob is recursive).

It might be good to add a comment showing what this is demonstrating (something like: "one pants target that owns sources in two subdirectories, and depends on another target for the third").